instrumentation/opentelemetry-instrumentation-system-metrics/tests/test_system_metrics.py

```diff
@@ -865,3 +866,14 @@ def test_open_file_descriptor_count(self, mock_process_num_fds):
             expected,
         )
         mock_process_num_fds.assert_called()
+
+
+class TestConfigSystemMetrics(TestBase):
+    # pylint:disable=no-self-use
+    def test_that_correct_config_is_read(self):
+        for key, value in _DEFAULT_CONFIG.items():
+            meter_provider = MeterProvider([InMemoryMetricReader()])
+            instrumentor = SystemMetricsInstrumentor(config={key: value})
+            instrumentor.instrument(meter_provider=meter_provider)
```

Comment:
Are there missing asserts here?

Reply:
Before the change a collection would have crashed because it would access keys that were not available in the config. The `# pylint:disable=no-self-use` is here because there are no asserts.